lib/test.py

```diff
@@ -350,7 +350,7 @@ def tap_parse_print_yaml(self, yml):
     def check_tap_output(self):
         """ Returns is_tap, is_ok, is_skip """
         try:
-            with open(self.tmp_result, 'r') as f:
+            with open(self.tmp_result, 'r', encoding='utf-8', errors='replace') as f:
```

Comment:
Looks like it deserves a separate commit.

Reply:
Moved it.